### ISSUE TITLE
Link failure with GraalVM 21.1.0 for Java 16

### ISSUE THREAD

**Issue by mipastgt** [gluonhq/gluonfx-maven-plugin#314]
I tested building my application with the client-maven-plugin on a Mac with the new GraalVM 21.1.0 for Java 16. It failed at the end with a link error.
```
[Do. Apr. 22 16:46:11 MESZ 2021][INFO] [SUB] ld: library not found for -lsunec
```
The exact same setup worked without problem with the same GraalVM version for Java 11. I used client-maven-plugin version 0.1.39, JavaFX 17-ea+7. The configuration was all created by running the agent.

**Comment by alialkubaisi** [comment]
Same here.

**Comment by markhuang1993** [comment]
I am build on Windows with same GraalVM and Java version, and have same error:

` [週一 7月 12 17:34:29 TST 2021][INFO] [SUB] LINK : fatal error LNK1181: 無法開啟輸入檔 'sunec.lib'`

This solve my problem:

1.Copy link command on error log
2.Remove 'sunec.lib' in command
3.Run new command again

**Comment by dwhitla** [comment]
I experienced the same issue today on MacOS 10.15.7 graalvm-ce-java16-21.2.0 using both gluonfx-maven-plugin 1.0.4 and 1.0.5-SNAPSHOT built from 9ff71e7

**Comment by tcsnzh** [comment]
I resolve it on Ubuntu. It's not a Mac, but it might help you

* first : find already existed `libsunec.a`

```shell
locate libsunec | grep libsunec.a
# /home/tcsnzh/桌面/packages/graalvm-ce-java11-linux-amd64-21.2.0/graalvm-ce-java11-21.2.0/lib/static/linux-amd64/glibc/libsunec.a
# /home/tcsnzh/桌面/packages/graalvm-ce-java11-linux-amd64-21.2.0/graalvm-ce-java11-21.2.0/lib/static/linux-amd64/musl/libsunec.a
# /home/tcsnzh/桌面/packages/graalvm-ce-java8-linux-amd64-21.2.0/graalvm-ce-java8-21.2.0/jre/lib/libsunec.a
```

* than , link it to system static lib path.

```shell
sudo ln -s /home/tcsnzh/桌面/packages/graalvm-ce-java11-linux-amd64-21.2.0/graalvm-ce-java11-21.2.0/lib/static/linux-amd64/musl/libsunec.a /usr/lib/libsunec.a
```

**Comment by jperedadnr** [comment]
sunec library has been removed from JDK 16+. The plugin (1.0.10) already checks if you are running on JDK 11 (keeps the library) or greater (removes the library).

**Comment by beatngu13** [comment]
@jperedadnr thank you for the update, I can confirm that it [works](https://github.com/beatngu13/pdf-zoom-wizard/actions/runs/1655765867) with 1.0.10 – at least on macOS and Windows. On Linux, I still get `[INFO] [SUB] /usr/bin/ld: cannot find -l:libsunec.a`. But this could also be an issue with my workflow, will dig a little deeper.

Side note: [1.0.1](https://github.com/gluonhq/gluonfx-maven-plugin/releases/tag/1.0.1) is still the latest release on GitHub, which might be confusing to some users.

**Comment by jperedadnr** [comment]
There was a small issue with sunec on Linux, it has been fixed with 1.0.11-SNAPSHOT
(Include this repo:)
```
<pluginRepositories>
        <pluginRepository>
            <id>Snapshots</id>
            <url>http://oss.sonatype.org/content/repositories/snapshots/</url>
        </pluginRepository>
    </pluginRepositories>
```

Yes, releases are done via Github actions, and they don't show as releases, but Maven badge shows 1.0.10

**Comment by tcsnzh** [comment]
> There was a small issue with sunec on Linux, it has been fixed with 1.0.11-SNAPSHOT (Include this repo:)
> 
> ```
> <pluginRepositories>
>         <pluginRepository>
>             <id>Snapshots</id>
>             <url>http://oss.sonatype.org/content/repositories/snapshots/</url>
>         </pluginRepository>
>     </pluginRepositories>
> ```
> 
> Yes, releases are done via Github actions, and they don't show as releases, but Maven badge shows 1.0.10

It require https default . so :

```
<pluginRepositories>
        <pluginRepository>
        <id>Snapshots</id>
        <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
    </pluginRepository>
</pluginRepositories>
 ```
